.github/docker/quick-lint-js-github-website-builder/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
```

Comment:
Must fix: Put `apt-get update` in a separate `RUN` from `apt-get install` like in our other `Dockerfile`s. (I know it might be against "best practice".)

Reply:
it will increase the size of the image though